docs/guides/guide-using-victoriametrics-to-monitor-vmware-vcenter.md

```diff
@@ -0,0 +1,617 @@
+chown root:root /usr/bin/vmagent-prod
+
+cat <<END >/etc/systemd/system/vmagent.service
+[Unit]
+Description=vmagent is a tiny agent which helps you collect metrics from various sources and store them in VictoriaMetrics.
+# https://docs.victoriametrics.com/vmagent.html
+After=network.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=simple
+User=victoriametrics
+Group=victoriametrics
+StartLimitBurst=5
+StartLimitInterval=0
+Restart=on-failure
+RestartSec=1
+EnvironmentFile=-/etc/victoriametrics/vmagent/vmagent.conf
+ExecStart=/usr/bin/vmagent-prod $ARGS
+ExecStop=/bin/kill -s SIGTERM $MAINPID
+ExecReload=/bin/kill -HUP $MAINPID
+# See docs https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#tuning
+LimitNOFILE=1048576
+LimitNPROC=1048576
+LimitCORE=infinity
+WorkingDirectory=/var/lib/vmagent-remotewrite-data
+ReadWritePaths=/var/lib/vmagent-remotewrite-data
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=vmagent
+PrivateTmp=yes
+ProtectHome=yes
+NoNewPrivileges=yes
+ProtectSystem=strict
+ProtectControlGroups=true
+ProtectKernelModules=true
+ProtectKernelTunables=yes
+END
+
+cat <<END >/etc/victoriametrics/vmagent/vmagent.conf
+# https://docs.victoriametrics.com/vmagent.html
+#
+# Example command line:
+# /path/to/vmagent -promscrape.config=/path/to/prometheus.yml -remoteWrite.url=https://victoria-metrics-host:8428/api/v1/write
+#
+# Please note that to write scraped data from vmagent to VictoriaMetrics Cluster you should use url like in this example -remoteWrite.url=http://vminsert-ip:8480/insert/0/prometheus/ .
+# See more information here https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#url-format.
+# 
+# If you only need to collect Influx data, then the following command is sufficient:
+#
+# /path/to/vmagent -remoteWrite.url=https://victoria-metrics-host:8428/api/v1/write
+#
+# Then send Influx data to http://vmagent-host:8429. See these https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#how-to-send-data-from-influxdb-compatible-agents-such-as-telegraf for more details.
+ARGS="-promscrape.config=/etc/victoriametrics/vmagent/scrape.yml -remoteWrite.url=http://127.0.0.1:8428/api/v1/write -remoteWrite.tmpDataPath=/var/lib/vmagent-remotewrite-data -promscrape.suppressScrapeErrors -influxTrimTimestamp=1s -remoteWrite.urlRelabelConfig=/etc/victoriametrics/vmagent/telegraf.yml"
+END
+
+cat <<END >/etc/victoriametrics/vmagent/scrape.yml
+# Scrape config example
+#
+global:
+  scrape_interval: 60s
+  scrape_timeout: 30s
+
+scrape_configs:
+  - job_name: vmagent
+    static_configs:
+    - targets: ['127.0.0.1:8429']
+  - job_name: victoria-metrics
+    static_configs:
+    - targets: ['127.0.0.1:8428']
+END
+
+# Start VictoriaMetrics
+systemctl enable vmagent.service
+systemctl restart vmagent.service
+```
+
+vmagent can be run with flags or specify to use parameters from variables. For the second option `/etc/victoriametrics/vmagent/vmagent.conf` will look this:
```

Comment:
> vmagent can be run with flags or specify to use parameters from variables

it is better to use language from vmagent docs here or place a link.

> For the second option

I think I missed the first option. What was it?